Tasks/BashV3/bash.ts

```diff
@@ -99,7 +99,7 @@ async function run() {
         tl.checkPath(tempDirectory, `${tempDirectory} (agent.tempDirectory)`);
         let fileName = uuidV4() + '.sh';
         let filePath = path.join(tempDirectory, fileName);
-        await fs.writeFileSync(
+        await fs.promises.writeFile(
```

Comment:
It is supported. However, we are in an `async` function and the code here seemed to be awaiting a sync call (which would do nothing) so I went ahead and updated it to the async version. It should have no affect switching between the two. 

I can move it back to the sync version if you think that would make more sense.

Reply:
Oh, right - didn't notice the await was there originally. I think the "normal" node way to do this would just be to call `fs.writeFileSync` without the await, not sure why we had that in the first place ¯\\_(ツ)_/¯

Reply:
I'd probably vote for just `fs.writeFileSync` instead of `await fs.promises.writeFile`, but its purely stylistic and I don't care too much. Your call - feel free to change it or just resolve this conversation.

Reply:
This task seems to be using `async/await` elsewhere so I am more of a fan of sticking with the async versions of fs calls than using the sync versions, so I'll keep it as is.